### PR TITLE
[ODS-5372] Support PS 7.2 and Unix - Script group 3

### PR DESCRIPTION
**Pull request by AxelMarquez**
Note that no changes were made to `database-management.psm1` since SQL Server for Linux is not going to be supported; PostgreSQL is the only supported engine in Linux.
To test `get-populated-from-nuget.ps1`, you have to use a v2 Nuget feed (v3 isn't supported by Save-Package), use: https://pkgs.dev.azure.com/ed-fi-alliance/Ed-Fi-Alliance-OSS/_packaging/EdFi/nuget/v2/

### CODE REVIEW

**Comment by bptillman** [comment]
Making a general comment here since the file was not change. The settings-teamcity.psm1 file was in the list of files to check, and on line 18 I believe we need to change `$env:TEAMCITY_BUILD_PROPERTIES_FILE` to `[Environment]::TEAMCITY_BUILD_PROPERTIES_FILE`

**Review by bptillman — CHANGES_REQUESTED** [review]
Lots of good work here in moving everything over. I did have some comments on things I think still need to change, but please review them and let me know if any don't look right to you.

File: logistics/scripts/modules/database/postgres-database-management.psm1 [hunk 1]
```diff
@@ -19,13 +19,8 @@ $script:packageVersion = "13.7.1"
...
-            return $true
-        }
-        catch {
-            return $false
-        }
+
+        return IsCommandAvailable "psql" "-V"
     }
 
     $packagePath = "$script:toolsPath/$script:packageName.$script:packageVersion"
```
[the remark this answers is not in the document]
Comment (AxelMarquez) [reply logistics/scripts/modules/database/postgres-database-management.psm1:26]: `postgres-database-management.psm1` isn't completely migrated yet, [ODS-5373](https://tracker.ed-fi.org/browse/ODS-5373) will complete its migration.

File: DatabaseTemplate/Modules/get-populated-from-web.ps1 [hunk 2]
```diff
@@ -3,6 +3,10 @@
 # The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
 # See the LICENSE and NOTICES files in the project root for more information.
 
+& "$PSScriptRoot/../../logistics/scripts/modules/load-path-resolver.ps1"
```
[the remark this answers is not in the document]
Comment (AxelMarquez) [reply DatabaseTemplate/Modules/get-populated-from-web.ps1]: Good catch, I moved them after the parameters but before the new use of Get-IsWindows

File: DatabaseTemplate/Modules/get-template-from-web.ps1 [hunk 3]
```diff
@@ -3,6 +3,10 @@
 # The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
 # See the LICENSE and NOTICES files in the project root for more information.
 
+& "$PSScriptRoot/../../logistics/scripts/modules/load-path-resolver.ps1"
```
[the remark this answers is not in the document]
Comment (AxelMarquez) [reply DatabaseTemplate/Modules/get-template-from-web.ps1]: Good catch, I moved them after the parameters but before the new use of Get-IsWindows

File: logistics/scripts/modules/utility/xml-validation.psm1 [hunk 4]
```diff
@@ -3,8 +3,8 @@
 # The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
 # See the LICENSE and NOTICES files in the project root for more information.
 
-& "$PSScriptRoot\..\..\..\..\logistics\scripts\modules\load-path-resolver.ps1"
-Import-Module -Force -Scope Global (Get-RepositoryResolvedPath 'logistics\scripts\modules\tasks\TaskHelper.psm1')
+& "$PSScriptRoot/../../../../logistics/scripts/modules/load-path-resolver.ps1"
+Import-Module -Force -Scope Global (Get-RepositoryResolvedPath 'logistics/scripts/modules/tasks/TaskHelper.psm1')
 
 function Get-XmlRoot {
     param(
```
[the remark this answers is not in the document]
Comment (AxelMarquez) [reply logistics/scripts/modules/utility/xml-validation.psm1:10]: I agree, I replaced backslashes with slashes in comments.

**Review by bptillman — APPROVED** [review]

**Pull request merged**